psi4/src/psi4/libmints/dimension.cc

```diff
@@ -66,14 +61,14 @@ void Dimension::print() const {
 }
 
 Dimension& Dimension::operator=(const int* other) {
-    for (int i = 0, maxi = n(); i < maxi; ++i) blocks_[i] = other[i];
+    for (size_t i = 0, maxi = n(); i < maxi; ++i) blocks_[i] = other[i];
```

Comment:
This shoud work:
```suggestion
    std::copy(other, other + n(), blocks_);
```

Reply:
Good idea, but as-is this leads to a build fail, for reasons I do not fully understand. My guess is that there is no suitable overload for handling `std::copy(T*, T*, std::vector<T>)`.
Getting the iterator of the destination with `blocks_.begin()` appears to make it work and pass the tests run by the CI.